tests/integration/aof.tcl

```diff
@@ -142,7 +142,7 @@ tags {"aof external:skip"} {
     ## Test that redis-check-aof indeed sees this AOF is not valid
     test "Short read: Utility should confirm the AOF is not valid" {
         catch {
-            exec src/redis-check-aof $aof_file
+            exec src/redis-check-aof $aof_manifest_file
```

Comment:
unrelated to this line. let's be sure to extend the tests to test all the AOF variations that we could be facing.
i.e. just test the happy path of:
1. single preamble aof with rdb head and aof resp tail
2. single old style resp aof (no rdb header)
3. manifest file with resp aof base
4. manifest file with rdb aof base
5. manifest file after upgrade, with base that's an rdb with aof tail

Reply:
OK，i will add this tests.